Makefile

```diff
@@ -12,6 +12,9 @@ build-all: embed_files lint build-amd64 build-arm64 build-arm32 build-windows bu
 build-all-small:
 	@$(MAKE) LDFLAGS+="-s -w" build-all
 
+build-releases:
+	@$(MAKE) LDFLAGS+="-s -w" -o lint embed_files build-all
```

Comment:
build-all is the same as build-all-small but building smaller binaries. Also, build-all (and build-all-small) both run line and embed_files so no need to include it here?

Reply:
This one does not run lint, because of `-o  lint`.

We don't run tests in that pipeline, why would we run all the things `lint` is doing? Well, except, to my confusion, `lint` actually runs:

1. `go fmt`, to format the code (I think that's provided by the Go version from the workflow).
1. `go mod tidy`, which will modify our go.mod (using the Go version from the workflow, not the one we tested earlier).
1. as a dependency of `vet`, everything that's in `embed_files`, which is needed for compiling.
1.  `go vet`, even though we don't run tests in this pipeline.

So my point is, most of those we don't want in this pipeline, if we want them, we probably want them in another order: generate, fmt, tidy, vet

Reply:
Oh I missed the `-o` part. Gotcha.